src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -73,13 +72,10 @@ export const useFormSend = () => {
           isClosable: true,
           position: 'top-right',
         })
-      } finally {
-        // Sends may be done from the context of a QR code modal, or a send modal, which are similar, but effectively diff. modal refs
-        qrCode.close()
-        send.close()
+        return ''
```

Comment:
q: I think we don't want this right?

Reply:
We do - this doesn't come as any difference from previous implementation, we were not re-throwing previously. But with the added `Promise<string>` return type, we now consistently return a string on every branch.

Since that can be confusing, changed the type to `string | undefined` and kept implicit undefined return as previously in f241cacf5b